src/main/java/core/module/ModuleManager.java

```diff
@@ -56,9 +55,7 @@ private void initMap(Map<Integer, IModule> map) {
 		map.put(Integer.valueOf(IModule.TSFORECAST), new TSForecastModule());
 		map.put(Integer.valueOf(IModule.SPECIALEVENTS), new SpecialEventsModule());
 		map.put(Integer.valueOf(IModule.TEAM_OF_THE_WEEK), new TeamOfTheWeekModule());
-		map.put(Integer.valueOf(IModule.EVIL_CARD), new EvilCardModule());
```

Comment:
Do we want to remove `EVIL_CARD` from `IModule` too?

Reply:
yes you are right